rpc/amqp-rpc.go

```diff
@@ -172,7 +169,7 @@ func (rpc *AmqpRpcServer) Start() (err error) {
 //
 // ```
 //   request = /* serialize request to []byte */
-//   response = <-AmqpRpcClient.Dispatch(method, request)
+//   response = <-AmqpRPCCLient.Dispatch(method, request)
```

Comment:
Sure. I didn't catch that one because golint doesn't know about AMQP being an acronym.